### DISCUSSION TITLE
Clarification on plugin version for Maya 2022

### DISCUSSION THREAD

**Discussion by tmafernando** [Autodesk/maya-usd#1366] [Q&A] [answered]
Hi everyone,

I'm still a bit confused about what's available in Maya 2022, and whether or not I need to do any extra installation myself.

Does the plugin that comes with Maya 2022 have the complete functionality of the open source plugin? or is it a partial release?

I'm looking to do some tests exporting USDSkel from Maya joints, as far as I know the original ALUsdMaya plugin had that capability, but I haven't been able to do it successfully.

Any help is appreciate, thanks!

**Comment by kxl-adsk** [comment, marked as answer]
MayaUSD can be released off Maya's release cycles. What this means is that we are free to release new versions at any cadence without being constrained by Maya releases.

Taking Maya 2022 as an example - we released MayaUSD 0.8.0 as part of the official Maya's installer. Later we made the 0.9.0 version available on GitHub with fixes and features that didn't make the original cut-off date for Maya's 2022 release.

I hope this clarifies the release process.

**Reply by tmafernando** [reply]
@kxl-adsk thanks for the information! That makes it clearer.
I was also able to find the version based on the directory in the Maya install directories.
